tools/noopfactory/src/test/resources/gen/NoOpAnyGenericInterface.kt

```diff
@@ -0,0 +1,15 @@
+package com.example
+
+import kotlin.Any
+import kotlin.Suppress
+import kotlin.collections.List
+import kotlin.collections.emptyList
+
+@Suppress(
+    "RedundantVisibilityModifier",
```

Comment:
Regarding this lint, can we disable it for the whole project ? IMHO it's not worthy of being a warning (especially since we might enable the [explicit API mode eventually](https://github.com/Kotlin/KEEP/blob/master/proposals/explicit-api-mode.md))

Reply:
hmmm we could do that yes.